icubam/backoffice/templates/icu.html

```diff
@@ -120,8 +130,23 @@ <h3 class="card-title">
 {% block scripts %}
 <script src="plugins/bootstrap-switch/js/bootstrap-switch.min.js"></script>
 <script>
-  $("input[data-bootstrap-switch]").each(function () {
-    $(this).bootstrapSwitch('state', $(this).prop('checked'));
+  $(function () {
+    $("input[data-bootstrap-switch]").each(function () {
+      $(this).bootstrapSwitch('state', $(this).prop('checked'));
+    })
+
+    function filInDataList (values, dataListId) {
+      var datalist = document.getElementById(dataListId);
+      values.forEach(function(item) {
+      var option = document.createElement('option');
+        option.value = item
+        datalist.appendChild(option)      
```

Comment:
The current version is fine, but just to confirm you could fill those options with Jinja2 templating as well right?

Reply:
absolutely. I was tempted to and to be honest i don t have a real reason to choose one rather than the other